data_juicer/ops/mapper/image_diffusion_mapper.py

```diff
@@ -0,0 +1,205 @@
+import copy
+import os
+
+from PIL import Image
+
+from data_juicer.utils.constant import Fields
+from data_juicer.utils.mm_utils import (SpecialTokens, load_data_with_context,
+                                        load_image, remove_special_tokens)
+from data_juicer.utils.model_utils import get_model, prepare_model
+
+from ..base_op import OPERATORS, Mapper
+from ..op_fusion import LOADED_IMAGES
+
+OP_NAME = 'image_diffusion_mapper'
+
```

Comment:
An AvailabilityChecking for this OP and its dependent OPs (GenerateCaptionMapper) might be required here.

Example:
https://github.com/alibaba/data-juicer/blob/77c755981de998729bb210f4bf2b91b765cbdb5a/data_juicer/ops/mapper/generate_caption_mapper.py#L21-L25

Reply:
Please check [pre-commit done](https://github.com/alibaba/data-juicer/pull/200/commits/beeb61a8f5ce5e5ae0770b0a64f74fc1e5b14adf)